style.css

```diff
@@ -0,0 +1,90 @@
+@import url("https://bermuda-css.netlify.app/style.css");
+@import url("./stylesheets/abstracts/colors.css");
+@import url("./stylesheets/utils/utility.css");
+
+body {
+  background-color: var(--defalult-bg-color);
+  box-sizing: border-box;
+  color: var(--default-text-color);
+}
+.root {
+  height: 100vh;
+  margin: auto;
+  width: 100vw;
+}
+/* content component */
+.content {
+  margin: auto;
+  position: relative;
+  width: 100%;
+}
+/* header comonent styling */
+.header-container {
+  position: sticky;
+  top: 0;
+  z-index: 999;
+}
+/* Navbar styling */
+.header__nav {
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
+  margin: auto;
+  padding: 0 var(--scale-md);
+}
+
+/* Nav Links Styling */
+.nav__links {
+  display: flex;
+}
+.nav__links li a {
+  color: var(--default-text-color);
+}
+.nav__links li a:hover {
+  border-bottom: 2px solid var(--primary-color);
+  box-sizing: border-box;
+  color: var(--primary-color);
+}
+
+/* Stling for nav__logo text */
+a h1,
+a h4 {
+  display: inline-block;
+}
+a .fa-store:hover {
+  transform: scale(1);
+}
```

Comment:
Use CSS styles to format the UI, instead of using random tags together.

Reply:
Refactored this code and fixed all semantic error